system-test/datastore.ts

```diff
@@ -268,6 +269,40 @@ describe('Datastore', () => {
       await datastore.delete(postKey);
     });
 
+    it('should save/get an int-able double value via Datastore.', async () => {
+      const postKey = datastore.key('Team');
+      const points = Datastore.double(2);
+      await datastore.save({key: postKey, data: {points}});
+      let [entity] = await datastore.get(postKey, {wrapNumbers: true});
```

Comment:
As a TODO, I wish we had unit tests for the Int and Double types, vs., only integration tests.

Reply:
I think a good unit test corresponding to this one would be to mock out the `get` and the `save` and ensure that getting a Double/Int leads to saving the same type.